--- a/src/third_party/vulkan-deps/vulkan-loader/src/loader/asm_offset.c
+++ b/src/third_party/vulkan-deps/vulkan-loader/src/loader/asm_offset.c
@@ -26,7 +26,7 @@
 #include "loader_common.h"
 #include "log.h"
 
-#if defined(__GNUC__) || defined(__clang__)
+#if (defined(__GNUC__) || defined(__clang__)) && !defined(__riscv)
 void produce_asm_define() {
     // GCC and clang make it easy to print easy to regex for values
     __asm__("# VULKAN_LOADER_ERROR_BIT = %c0" : : "i"(VULKAN_LOADER_ERROR_BIT));
