--- a/src/electron/script/lib/get-version.js
+++ b/src/electron/script/lib/get-version.js
@@ -6,6 +6,7 @@
 
 // Print the value of electron_version set in gn config.
 module.exports.getElectronVersion = () => {
+  return '29.4.2';
   // Read the override_electron_version from args.gn file.
   try {
     const outDir = path.resolve(ELECTRON_DIR, '..', 'out', getOutDir());
--- a/src/electron/BUILD.gn
+++ b/src/electron/BUILD.gn
@@ -107,21 +107,7 @@
 if (override_electron_version != "") {
   electron_version = override_electron_version
 } else {
-  # When building from source code tarball there is no git tag available and
-  # builders must explicitly pass override_electron_version in gn args.
-  # This read_file call will assert if there is no git information, without it
-  # gn will generate a malformed build configuration and ninja will get into
-  # infinite loop.
-  read_file(".git/packed-refs", "string")
-
-  # Set electron version from git tag.
-  electron_version = exec_script("script/get-git-version.py",
-                                 [],
-                                 "trim string",
-                                 [
-                                   ".git/packed-refs",
-                                   ".git/HEAD",
-                                 ])
+  electron_version = "29.4.2"
 }
 
 if (is_mas_build) {
